CHANGELOG.md

```diff
@@ -1,6 +1,12 @@
 Changelog
 =========
 
+2.4 - January 21th, 2018
```

Comment:
Remove the release number + date heading. It will be added later when your code is actually released.